webapp/components/interop.html

```diff
@@ -333,6 +343,13 @@
       return metadata && metadata.test_runs;
     }
 
+    computeTestPaths(fileMetrics) {
+      if (!fileMetrics) {
+        return [];
+      }
+      return new Set(fileMetrics.map(m => m.dir));
```

Comment:
nit: We use `reduce()` instead in the other `computeTestPaths` which is more lines of code, but has the advantage of not building an array for nearly immediate garbage collection. We should pick one strategy for both implementations.

Reply:
Consolidated to use `Set` ctor